etna/media/models.py

```diff
@@ -1,16 +1,38 @@
 import mimetypes
 
 from django.conf import settings
+from django.core.validators import MinValueValidator
 from django.db import models
 
+from wagtail import blocks
 from wagtail.api import APIField
-from wagtail.fields import RichTextField
+from wagtail.fields import RichTextField, StreamField
 
 from wagtailmedia.models import AbstractMedia
 
+from etna.core.blocks.paragraph import APIRichTextBlock
 from etna.core.serializers import RichTextSerializer
 
 
+class MediaChapterSectionBlock(blocks.StructBlock):
+    time = blocks.DecimalBlock(
+        blank=True,
+        default=0,
+        validators=[MinValueValidator(0)],
+        label="Time in seconds",
+    )
+    heading = blocks.CharBlock(max_length=20)
+    transcript = APIRichTextBlock(required=False, features=["bold", "italic"])
+
+    def get_api_representation(self, value, context=None):
+        representation = super().get_api_representation(value, context)
+        representation["time"] = int(representation["time"])
```

Comment:
I couldn't figure out how to make `DecimalBlock` only store integers - it always wanted to store one decimal place.

Reply:
`IntegerBlock` will solve the decimal thing but for some reason it's saved as a string so you'll still need to wrap it in `int()`, up to you if you want Decimal or Integer block!